src/app/components/Label.tsx

```diff
@@ -4,12 +4,17 @@ import { red } from '../../styles/colors'
 import { Regular } from '../../styles/components/Text'
 import { styled } from 'linaria/react'
 
-const RedLabel = styled.label`
+interface WarningLabelProp {
+  isBlocked: boolean
+}
+
+const Label = styled.label<WarningLabelProp>`
   ${Regular}
-  color: ${red};
   padding-bottom: 1rem;
+  transition: color 0.5s;
+  color: ${props => (props.isBlocked ? red : 'transparent')};
```

Comment:
```suggestion
  transition: visibility 0.5s;
  visibility: ${props => (props.isBlocked ? 'visible' : 'hidden')};
  color: ${red};
```

보다 시멘틱적인 스타일을 위해 `visibility: hidden`를 사용한 더미 레이블을 고려했으나 transition 속성으로 `visibility`가 유효하지 않았습니다.

Reply:
그러면 opacity를 이용하면 어떨까요?